stork-quickstart/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Could you reformat with 4 spaces? That's what we're trying to do with your POMs.